internal/southboundhandler/handler.go

```diff
@@ -354,27 +381,14 @@ func extractBootstrapScript(secret *corev1.Secret, kind, providerID string) (str
 	if err != nil {
 		return "", err
 	}
-	switch {
-	case kind == "KubeadmConfig":
+	switch kind {
```

Comment:
This is not a blocking comment, but I think we should eventually move all these switches into factories (based on the provider we get the install and uninstall commands). I know we don't have time this sprint so I would maybe add it to the backlog as an improvement

Reply:
I created a JIRA ticket for that task